pkgs/development/libraries/pipewire/default.nix

```diff
@@ -125,7 +128,7 @@ let
 
     mesonFlags = [
       "-Ddocs=enabled"
-      "-Dman=disabled" # we don't have xmltoman
+      "-Dman=enabled"
```

Comment:
This line can be dropped, we enable autofeatures by default.

Reply:
(done)